app/components/ProductList/index.js

```diff
@@ -50,27 +54,31 @@ class ProductList extends React.Component {
     this.setState({ 
       sortBy: value, 
       orderDirection: this.state.orderDirection,
```

Comment:
You don't need to set orderDirection here, as it doesn't change. setState will only update the keys you pass to it (in this case this.setState({ sortBy: value }) will only update sortBy but not delete orderDirection